test/integration/model/BpmnModdleSpec.js

```diff
@@ -22,11 +22,9 @@ describe('bpmn-moddle', function() {
         '</bpmn2:definitions>';
 
       // when
-      parse(xml, function(err, definitions) {
+      parse(xml).then(function(result) {
```

Comment:
Again, we can return the whole promise and if the `expect`ations are not met, promise rejection will cause the test to fail.